opentelemetry-api/src/opentelemetry/resources/__init__.py

```diff
@@ -11,3 +11,45 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from abc import ABC, abstractmethod
+from typing import Dict, Union
+
+
+class Resource(ABC):
+    """This the interface that resources must implement"""
```

Comment:
"This the" -> "This is the"?

Reply:
fixed. sorry I'll scan for grammatical errors myself as well.